### ISSUE TITLE
FIX - cross browser interneationalization

### ISSUE THREAD

**Issue by ianbogda** [FileZ/FileZ#72]
Chromium doesn't care about 
Zend_Locale::setDefault (fz_config_get ('app', 'default_locale', 'fr'));
$currentLocale = new Zend_Locale ('auto');

```
diff --git a/index.php b/index.php
index 51ca3f0..5a4748d 100644
--- a/index.php
+++ b/index.php
@@ -105,8 +105,8 @@ function before () {
     }

     // I18N
-    Zend_Locale::setDefault (fz_config_get ('app', 'default_locale', 'fr'));
-    $currentLocale = new Zend_Locale ('auto');
+    $currentLocale = new Zend_Locale (fz_config_get ('app', 'default_locale', 'fr'));
     $translate     = new Zend_Translate ('gettext', option ('root_dir').DIRECTORY_SEPARATOR.'i18n', $currentLocale,
         array('scan' => Zend_Translate::LOCALE_DIRECTORY));
     option ('translate', $translate);
```

**Comment by urbie-mk2** [comment]
could you edit your post to apply quoting, please ?
use 3 apostrophes to quote

```
like this

```

becomes


```

this

```

read more about it here: 
http://github.github.com/github-flavored-markdown/
```

**Comment by almaghi** [comment]
thank you both of you. 
Fixed with 33052202bb9adc7131382930555146d2b7ee1e8a
